frontend/src/components/Notificacao/Evolucao/Cons.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div>
+    <v-row>
+      <v-col>
+        <h4 class="primary--text font-weight-medium title">IDENTIFICAÇÃO DO NOTIFICANTE</h4>
+        <v-row dense>
+          <v-col>
+            <strong>Nome</strong>
+            <p>{{notificacao.nome}}</p>
+          </v-col>
+          <v-col>
+            <strong>Documento</strong>
+            <p>{{notificacao.documento}}</p>
+          </v-col>
+          <v-col>
+            <strong>Telefone</strong>
```

Comment:
Mesmo detalhe do negrito
```suggestion
            <span class="font-weight-bold">Telefone</span>
```